code/modules/mob/living/carbon/xenomorph/Abilities.dm

```diff
@@ -142,7 +142,7 @@
 
 	playsound(xeno.loc, pick(xeno.screech_sound_effect_list), 75, 0, status = 0)
 	xeno.visible_message(SPAN_XENOHIGHDANGER("[xeno] emits an ear-splitting guttural roar!"))
-	xeno.create_shriekwave() //Adds the visual effect. Wom wom wom
+	xeno.create_shriekwave(xeno, 15) //Adds the visual effect. Wom wom wom, 15 shriekwaves
```

Comment:
You pass the xeno here but don't actually use it in the proc

Reply:
Refactor it out - you can already tell what xeno is shieking by looking at src